user-guide/Troubleshooting/Known_issues/KI_Upgrade_fails_VerifyNoLegacyReportsDashboards_prerequisite.md

```diff
@@ -0,0 +1,23 @@
+---
+uid: KI_Upgrade_fails_VerifyNoLegacyReportsDashboards_prerequisite
+---
+
+# Upgrade fails because of VerifyNoLegacyReportsDashboards.dll prerequisite
+
+## Affected versions
+
+DataMiner 10.4.0 or newer.<!-- RN 37786 -->
+
+## Cause
+
+From DataMiner 10.4.0 onwards, the [legacy Reporter](xref:reporter) and [legacy Dashboards](xref:dashboards) modules are *End of Life*. These have been replaced by the [Dashboards app](xref:newR_D). The *VerifyNoLegacyReportsDashboards* upgrade prerequisite check verifies that no legacy reports and legacy dashboards still exist before upgrading, as these will no longer work after the upgrade.
+
+## Fix
+
+If you no longer need the legacy Reporter and Dashboards modules, remove any existing legacy reports and legacy dashboards by deleting them in the Reporter and Dashboards modules. Note that other DataMiner modules, such as Automation and Scheduler, may also make use of these.
+
+If you still rely on certain functionality and want to keep on using the legacy Reporter and Dashboards modules, set the [*LegacyReportsAndDashboards* soft-launch option](xref:Overview_of_Soft_Launch_Options#legacyreportsanddashboards) to *true*, then run `C:\Skyline DataMiner\Tools\ConfigureIIS.bat` as Administrator, and restart the DataMiner Agent.
```

Comment:
Is running C:\Skyline DataMiner\Tools\ConfigureIIS.bat as Administrator and restarting the DMA always necessary when you enable this option? Because if so, I think we'd best also add this info on the soft-launch options page.

Reply:
For the `LegacyReportsAndDashboards` and the `LegacyAnnotations` soft launch flags, that's indeed required. Not for the others. This is also mentioned on the hardening guide page.